Composer/packages/tools/language-servers/language-understanding/src/LUServer.ts

```diff
@@ -243,6 +245,38 @@ export class LUServer {
     this.LUDocuments.push(luDocument);
   }
 
+  protected async findAllImportedEntities(
+    imports: { id: string; path: string; description: string }[],
+    importResolver: (source: string, id: string) => Promise<{ id: string; content: any }>,
+    luFeatures: any
+  ): Promise<string[]> {
+    let content = '';
+    const result: string[] = [];
+    for (const importFile of imports) {
+      try {
+        content = (await importResolver('.', importFile.id)).content;
+      } catch (error) {
+        // ignore if file not exist
+      }
+
+      const { resource } = await this.luParser.parse(content, importFile.id, luFeatures);
+      const sections = resource.Sections;
+      for (const section of sections) {
+        if (section.SectionType === 'newEntitySection') {
```

Comment:
>newEntitySection [](http://example.com/codeflow?start=37&length=16)

minor: looks like these strings are also defined here, maybe we can use them? 

<[https://github.com/microsoft/botframework-cli/blob/7e5fa93174a86f61467d6fc3c867d0cf36d8d8c3/packages/lu/src/parser/utils/enums/lusectiontypes.js](https://github.com/microsoft/botframework-cli/blob/7e5fa93174a86f61467d6fc3c867d0cf36d8d8c3/packages/lu/src/parser/utils/enums/lusectiontypes.js)
>

Reply:
It is exported outside bf-lu package. I defined some constants for these string values in LuServer.